baseapp-profiles/baseapp_profiles/permissions.py

```
@@ -45,3 +47,12 @@ def has_perm(self, user_obj, perm, obj=None):
                     or user_obj.is_superuser
                     or obj.members.filter(user_id=user_obj.id).exists()
                 )
+
+        if perm == "baseapp_profiles.change_profileuserrole" and obj:
+            if isinstance(obj, Profile):
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

(SIM102)